package/MDAnalysis/analysis/contacts.py

```diff
@@ -386,15 +386,18 @@ class Contacts(AnalysisBase):
     .. versionchanged:: 2.0.0
        :attr:`timeseries` results are now stored in a
        :class:`MDAnalysis.analysis.base.Results` instance.
+    .. versionchanged:: 2.2.0
+       :class: `Contacts` accepts both AtomGroup and string for `select`
```

Comment:
Unfortunately read the docs hasn't triggered but there shouldn't be a blank space between :class: and the classname.
```suggestion
       :class:`Contacts` accepts both AtomGroup and string for `select`
```